boards/arm/nucleo_h745zi_q/Kconfig.defconfig

```diff
@@ -0,0 +1,55 @@
+# STM32H745ZI Nucleo board configuration
+
+# Copyright (c) 2020 Alexander Kozhinov
+# AlexanderKozhinov@yandex.com
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NUCLEO_H745ZI_Q_M7 || BOARD_NUCLEO_H745ZI_Q_M4
+
+config BOARD
+	default "nucleo_h745zi_q_m7" if BOARD_NUCLEO_H745ZI_Q_M7
+	default "nucleo_h745zi_q_m4" if BOARD_NUCLEO_H745ZI_Q_M4
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 400000000 if BOARD_NUCLEO_H745ZI_Q_M7
+	# SYS_CLOCK_M4 = SYS_CLOCK_M7 / CLOCK_STM32_HPRE
+	default 200000000 if BOARD_NUCLEO_H745ZI_Q_M4
+
+config CLOCK_STM32_D1CPRE
+	default 1
```

Comment:
I think all these values should go into defconfig

Reply:
Do you mean nucleo_h745zi_q_m7_defconfig ?

Reply:
added support for cpu1.

Reply:
This will stay better there, because they are for the SoC and not for M4/M7. Any other suggestions?